{"name": "test_student_registration_form", "status": "failed", "statusDetails": {"message": "selene.core.exceptions.TimeoutException: Message: \n\nTimed out after 6.0s, while waiting for:\nbrowser.all(('css selector', '.table-dark>tbody>tr>td:nth-child(2)')).has texts ('Имя Фамилия', 'testmail@mail.gg', 'Other', '2589632147', 'Physics', 'Reading', 'hedgehog.jpg', '221b, Baker Street, London, NW1 6XE, UK', 'NCR Delhi')\n\nReason: AssertionError: actual visible_texts: ['Имя Фамилия', 'testmail@mail.gg', 'Other', '2589632147', '22 January,2024', 'Physics', 'Reading', 'hedgehog.jpg', '221b, Baker Street, London, NW1 6XE, UK', 'NCR Delhi']\nScreenshot: file://C:\\Users\\asya4\\.selene\\screenshots\\1705936322210\\1705936322210.png\nPageSource: file://C:\\Users\\asya4\\.selene\\screenshots\\1705936322210\\1705936322210.html", "trace": "fn = <selene.core.conditions.CollectionCondition object at 0x000001A0E0D936D0>\n\n    def logic(fn: Callable[[E], R]) -> R:\n        finish_time = time.time() + self._timeout\n    \n        while True:\n            try:\n>               return fn(self.entity)\n\n..\\..\\.venv\\Lib\\site-packages\\selene\\core\\wait.py:119: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n..\\..\\.venv\\Lib\\site-packages\\selene\\core\\condition.py:174: in __call__\n    return self._fn(entity)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nentity = <selene.core.entity.Collection object at 0x000001A0E0166D90>\n\n    def fn(entity: E) -> None:\n        query_to_str = str(query)\n        result = (\n            query.__name__ if query_to_str.startswith('<function') else query_to_str\n        )\n        actual = query(entity)\n        if not predicate(actual):\n>           raise AssertionError(f'actual {result}: {actual}')\nE           AssertionError: actual visible_texts: ['Имя Фамилия', 'testmail@mail.gg', 'Other', '2589632147', '22 January,2024', 'Physics', 'Reading', 'hedgehog.jpg', '221b, Baker Street, London, NW1 6XE, UK', 'NCR Delhi']\n\n..\\..\\.venv\\Lib\\site-packages\\selene\\core\\condition.py:146: AssertionError\n\nDuring handling of the above exception, another exception occurred:\n\nsetup_browser = <selene.core.entity.Browser object at 0x000001A0E0D50CD0>\n\n    @allure.tag(\"web\")\n    @allure.severity(Severity.BLOCKER)\n    @allure.label(\"owner\", \"as.shemetova\")\n    @allure.description(\"Registration\")\n    @allure.feature(\"Registration\")\n    def test_student_registration_form(setup_browser):\n        with allure.step(\"Открыть страницу регистрации пользователей\"):\n            practice_form.open()\n        with allure.step(\"Заполнить форму регистрации тестовыми данными\"):\n            user = User(first_name='Имя',\n                        last_name='Фамилия',\n                        email='testmail@mail.gg',\n                        gender='Other',\n                        # month_of_birth='November',\n                        phone_number='2589632147',\n                        # year_of_birth='2006',\n                        # day_of_birth='19',\n                        subject='Physics',\n                        hobby='Reading',\n                        image='hedgehog.jpg',\n                        address='221b, Baker Street, London, NW1 6XE, UK',\n                        state='NCR',\n                        city='Delhi')\n            practice_form.registration(user)\n        with allure.step(\"Проверка, что пользователь зарегистрирован\"):\n>           practice_form.student_should_be_registred(user)\n\ntest_registration_form.py:36: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n..\\page_object\\registration_page.py:38: in student_should_be_registred\n    browser.all(\".table-dark>tbody>tr>td:nth-child(2)\").should(have.texts(\n..\\..\\.venv\\Lib\\site-packages\\selene\\core\\entity.py:136: in should\n    self.wait.for_(condition)\n..\\..\\.venv\\Lib\\site-packages\\selene\\core\\wait.py:143: in for_\n    return self._decorator(self)(logic)(fn)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nfn = <selene.core.conditions.CollectionCondition object at 0x000001A0E0D936D0>\n\n    def logic(fn: Callable[[E], R]) -> R:\n        finish_time = time.time() + self._timeout\n    \n        while True:\n            try:\n                return fn(self.entity)\n            except Exception as reason:\n                if time.time() > finish_time:\n                    reason_message = str(reason)\n    \n                    reason_string = '{name}: {message}'.format(\n                        name=reason.__class__.__name__,\n                        message=reason_message,\n                    )\n                    # TODO: think on how can we improve logging failures in selene, e.g. reverse msg and stacktrace\n                    # stacktrace = getattr(reason, 'stacktrace', None)\n                    timeout = self._timeout\n                    entity = self.entity\n    \n                    failure = TimeoutException(\n                        f'\\n'\n                        f'\\nTimed out after {timeout}s, while waiting for:'\n                        f'\\n{entity}.{fn}'\n                        f'\\n'\n                        f'\\nReason: {reason_string}'\n                    )\n    \n>                   raise self._hook_failure(failure)\nE                   selene.core.exceptions.TimeoutException: Message: \nE                   \nE                   Timed out after 6.0s, while waiting for:\nE                   browser.all(('css selector', '.table-dark>tbody>tr>td:nth-child(2)')).has texts ('Имя Фамилия', 'testmail@mail.gg', 'Other', '2589632147', 'Physics', 'Reading', 'hedgehog.jpg', '221b, Baker Street, London, NW1 6XE, UK', 'NCR Delhi')\nE                   \nE                   Reason: AssertionError: actual visible_texts: ['Имя Фамилия', 'testmail@mail.gg', 'Other', '2589632147', '22 January,2024', 'Physics', 'Reading', 'hedgehog.jpg', '221b, Baker Street, London, NW1 6XE, UK', 'NCR Delhi']\nE                   Screenshot: file://C:\\Users\\asya4\\.selene\\screenshots\\1705936322210\\1705936322210.png\nE                   PageSource: file://C:\\Users\\asya4\\.selene\\screenshots\\1705936322210\\1705936322210.html\n\n..\\..\\.venv\\Lib\\site-packages\\selene\\core\\wait.py:141: TimeoutException"}, "description": "Registration", "steps": [{"name": "Открыть страницу регистрации пользователей", "status": "passed", "start": 1705936330112, "stop": 1705936332719}, {"name": "Заполнить форму регистрации тестовыми данными", "status": "passed", "start": 1705936332719, "stop": 1705936337437}, {"name": "Проверка, что пользователь зарегистрирован", "status": "failed", "statusDetails": {"message": "selene.core.exceptions.TimeoutException: Message: \n\nTimed out after 6.0s, while waiting for:\nbrowser.all(('css selector', '.table-dark>tbody>tr>td:nth-child(2)')).has texts ('Имя Фамилия', 'testmail@mail.gg', 'Other', '2589632147', 'Physics', 'Reading', 'hedgehog.jpg', '221b, Baker Street, London, NW1 6XE, UK', 'NCR Delhi')\n\nReason: AssertionError: actual visible_texts: ['Имя Фамилия', 'testmail@mail.gg', 'Other', '2589632147', '22 January,2024', 'Physics', 'Reading', 'hedgehog.jpg', '221b, Baker Street, London, NW1 6XE, UK', 'NCR Delhi']\nScreenshot: file://C:\\Users\\asya4\\.selene\\screenshots\\1705936322210\\1705936322210.png\nPageSource: file://C:\\Users\\asya4\\.selene\\screenshots\\1705936322210\\1705936322210.html\n\n", "trace": "  File \"C:\\Users\\asya4\\PycharmProjects\\diploma_project\\ui\\tests\\test_registration_form.py\", line 36, in test_student_registration_form\n    practice_form.student_should_be_registred(user)\n  File \"C:\\Users\\asya4\\PycharmProjects\\diploma_project\\ui\\page_object\\registration_page.py\", line 38, in student_should_be_registred\n    browser.all(\".table-dark>tbody>tr>td:nth-child(2)\").should(have.texts(\n  File \"C:\\Users\\asya4\\PycharmProjects\\diploma_project\\.venv\\Lib\\site-packages\\selene\\core\\entity.py\", line 136, in should\n    self.wait.for_(condition)\n  File \"C:\\Users\\asya4\\PycharmProjects\\diploma_project\\.venv\\Lib\\site-packages\\selene\\core\\wait.py\", line 143, in for_\n    return self._decorator(self)(logic)(fn)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\asya4\\PycharmProjects\\diploma_project\\.venv\\Lib\\site-packages\\selene\\core\\wait.py\", line 141, in logic\n    raise self._hook_failure(failure)\n"}, "start": 1705936337437, "stop": 1705936346201}], "start": 1705936330112, "stop": 1705936346213, "uuid": "5c8d3907-7999-4d0c-86ee-017f9faedcca", "historyId": "049d9c54195a75b294f8d995b7392a37", "testCaseId": "517aaecb6507ba288982d1eecc94ca86", "fullName": "ui.tests.test_registration_form#test_student_registration_form", "labels": [{"name": "feature", "value": "Registration"}, {"name": "severity", "value": "blocker"}, {"name": "owner", "value": "as.shemetova"}, {"name": "tag", "value": "web"}, {"name": "parentSuite", "value": "ui.tests"}, {"name": "suite", "value": "test_registration_form"}, {"name": "host", "value": "MSI"}, {"name": "thread", "value": "8140-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "ui.tests.test_registration_form"}]}